airflow/providers/hashicorp/secrets/vault.py

```diff
@@ -40,10 +40,10 @@ class VaultSecrets(BaseSecretsBackend, LoggingMixin):
 
         [secrets]
         backend = airflow.providers.hashicorp.secrets.vault.VaultSecrets
-        backend_kwargs = {"path":"connections","url":"http://127.0.0.1:8200","mount_point":"airflow"}
+        backend_kwargs = {"connections_path":"conns","url":"http://127.0.0.1:8200","mount_point":"airflow"}
 
-    For example, if your keys are under ``connections`` path in ``airflow`` mount_point, this
-    would be accessible if you provide ``{"path": "connections"}`` and request
+    For example, if your keys are under ``conns`` path in ``airflow`` mount_point, this
+    would be accessible if you provide ``{"connections_path": "conns"}`` and request
     conn_id ``smtp_default``.
```

Comment:
@turbaszek @kaxil I changed `connections` to `conns` in the docstring to reduce the line length < 110.

Reply:
Can't we use `connections` and just break the lines?

Reply:
I would prefer `connections_path` too, please

Reply:
May be I should have asked this before. For some reason, I was under the impression that we can not break lines in the .cfg file. So can we break lines?

Something like this.
```
backend_kwargs = 
        {"connections_path":"connections","url":"http://127.0.0.1:8200","mount_point":"airflow"}

# or
backend_kwargs = {"connections_path":"connections",
                  "url":"http://127.0.0.1:8200",
                  "mount_point":"airflow"}
```

Reply:
Multiline config are handled fine as long as they are indented.

Example:

```ini
[Multiline Values]
chorus: I'm a lumberjack, and I'm okay
    I sleep all night and I work all-day
```

I am also fine for you to disable long-line check for this particular case so it is easy for users to copy/paste.

Reply:
Got it! I just updated the docstring with multiline values.